### ISSUE TITLE
Prepare Stackless 3.7

### ISSUE THREAD

**Issue by akruis** [stackless-dev/stackless#149]
Time to prepare Stackless 3.7.
1. Merge branch 'master' into 'master-slp' up to commit 1e17d4aaff5.
2. Start branch 3.7-slp
3. Merge branch '3.7' into '3.7-slp' up to v3.7.0

**Comment by akruis** [comment]
I'm going to change the "Stackless version" to "3.7". Keeping it at "3.1b3 060516" makes no sense at all.

**Comment by akruis** [comment]
I'm mostly ready with the preparation of Stackless 3.7. Still to do:

- [ ] Merge pull request #198 
- [ ] Merge pull request #188
- [ ] Merge pull request #197 
- [ ] Merge pull request #199 
- [ ] Merge pull request #192 
- [ ] Merge pull request #201 
- [ ] Merge pull request #202 
- [ ] Merge pull request #203 
- [ ] Create branch "3.7-slp"
- [ ] Merge upstream commits from branch "3.7". For the largest part this is already done in https://github.com/akruis/cpython/tree/slp37

Once this is done, I can prepare a release.

**Comment by akruis** [comment]
I just pushed 3.7-slp with tags v3.7.0-slp and v3.7.1-slp. The tests of v3.7.2-slp are still running.

**Comment by akruis** [comment]
Source code of v3.7.2-slp is available on stackless.com.

Within the next days I'll build the Windows binaries and the Conda versions.

**Comment by akruis** [comment]
Windows artefacts are available on Stackless.com.

**Comment by akruis** [comment]
Conda packages are available from https://anaconda.org/stackless (Windows and Linux 64bit).
